api/Element.json

```diff
@@ -6590,7 +6591,8 @@
               }
             ],
             "edge": {
-              "version_added": "13"
+              "version_added": "13",
+              "notes": "From version 92, returns a promise instead of <code>undefined</code>. The behavior reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
```

Comment:
```suggestion
              "notes": "From version 92, returns a promise instead of <code>undefined</code>. The behavior reflects <a href='https://github.com/w3c/pointerlock/pull/49'>a proposed specification change</a>."
```